docs/lesson/ontology-term-use.md

```diff
@@ -121,7 +121,7 @@ The figure below by [Chris Mungall](https://bids.berkeley.edu/people/chris-munga
 <a name="request"></a> 
 ## 7. Make term requests to existing ontologies
 
-See separate lesson on [Making term requests to existing ontologies](../how-to/term-request.md).
+See separate lesson on [Making term requests to existing ontologies](https://oboacademy.github.io/obook/howto/term-request/).

```

Comment:
Always use relative links when linking to other obook pages `../howto/term-request.md`

Reply:
will do, thanks!